.github/workflows/compile.yml

```diff
@@ -28,6 +28,8 @@ jobs:
       fail-fast: false
       matrix:
         os:
+        - macos-11  # Intel
+        - macos-12  # Intel
```

Comment:
macos-11 is already dead and 12 will be removed soon
- https://github.com/actions/runner-images/issues/10721

Reply:
Thanks information! I didn't know that.
It's tough to lose the means of detecting the problems that occurred this time.